README.org

```diff
@@ -69,6 +65,22 @@ There is an option to have your user credentials (email address and password) sa
 The default is not to do this because if not properly configured it would save these unencrypted which is not a good default to have.
 Customize the variable =mastodon-auth-source-file= if you want to enable this feature.
 
+*** 2 Factor Auth
+**** Overview
+After setting up =mastodon-instance-url= set the following in your
+=.emacs= or =customize=:
+
+1. =mastodon-auth-mechanism=
+
+**** Set up
+#+BEGIN_SRC emacs-lisp
+  (setq mastodon-auth-mechanism "oauth2")
+#+END_SRC
+
+The OAuth2 library uses =plstore= to securely store the credentials.
```

Comment:
Can you add a link somewhere appropriate to the oauth2 library (plus maybe a note which version)?

E.g. I don't use melpa, so I'll need to find the library and manually add it to me config files repo.

Reply:
oauth2.el is available on elpa. Do you think we should link it?

Reply:
BTW: I found and use https://github.com/emacsmirror/oauth2.git in my own setup.